java-squid/src/main/java/org/sonar/java/se/ExplodedGraphWalker.java

```diff
@@ -574,6 +575,17 @@ private static boolean isLocalMethodInvocation(MethodInvocationTree tree) {
     return false;
   }
 
+  public static boolean isObjectsMethod(MethodInvocationTree syntaxNode, String methodName, int numberOfParameter) {
+    Symbol symbol = syntaxNode.symbol();
+    if (symbol.isMethodSymbol()) {
+      Symbol.MethodSymbol methodSymbol = (Symbol.MethodSymbol) symbol;
+      if (methodSymbol.owner().type().is("java.util.Objects") && methodName.equals(methodSymbol.name())) {
+        return methodSymbol.parameterTypes().size() == numberOfParameter;
```

Comment:
handling of number of parameters is useless at this point and can be handled separately.